### ISSUE TITLE
Export function for measurements

### ISSUE THREAD

**Issue by nyukyi** [spawaskar-cora/cora-docs#412]
I'm working on making pair match lists for some USS Oklahoma Group 2 elements and realized that there is no feature/function to export the measurements for the elements.  It would be super nice to have that function so that analysts wouldn't have to type in measurements manually onto an excel sheet for the pair match list making process.

**Comment by emilystreetman** [comment]
Per scrum today, we believe this functionality exists in the Reports>Measurement Report function. Measurements would be exportable then, although the analyst would still have to manually select which elements from the downloaded xlsx would be used in the comparison. For full automation of a pair match list, the ongoing efforts to link CoRA with CoRA Analytics are in the works but not operational yet. If this is sufficient for now, suggest closing ticket.

I am now reading this ticket differently than how we read it in scrum. @cbrown311, what about this?: I suggest that @nyukyi wants to do Specimens >Reports>Measurements Report>Provenance 1>G-02 and export the measurements for only G-02 elements. This search criteria returns "results" but no measurements. See attached screenshot.

![image](https://user-images.githubusercontent.com/34662044/55589965-48cfce80-56f7-11e9-880f-8f5dbcba5b34.png)

**Comment by fedamann** [comment]
@emilystreetman @cbrown311 I agree that it would be a nice feature enhancement to return a measurement report for any one of the Report Criteria listed.  In the example however, @nyukyi can still pull the list of measurements for pair-matching because she will know what specimen type to pull from for the comparisons.  The suck part is that if she has to pair match a femur, humerus, radius, tibia... then she has to run as many reports against Group-2 to pull the measurement data for xls export.  The other issue is the results returned from multiple specimen types would make the report results rather messy and this is probably best left for a full export function to run in the back-end and then have it emailed to you once the job is complete with the tab or csv export file.  I recommend we move to backlog and wait until @SachinPawaskarUNO gets the export function and the jobs and email server running.

**Comment by clegarde** [comment]
Looking at @emilystreetman comment and image - it looks like if you do not choose a bone, it will only return what looks like an inventory list. Bone is marked with an * but you can still run the report without choosing one, which gives you the inventory list, not measurements. If you choose G-02 and a bone, then you get measurements. Or if you choose only a bone or Individual Number you get a measurement list that you can export.

Since this report is currently set to only give results by bone or Individual Number, not by only Accession, Prov1, or Prov2, it might make more sense to users if CoRA requires you to choose a bone or Individual Number (reflected by the *) to run the report. For example, if you choose only G-02 and press "Generate" it would return a message that you must choose a Bone or Individual Number. Just to make the current reporting function clear for users. I agree with @fedamann that the goal would be to have the ability to run a report by any of the criteria (Accession, Prov1, Prov2, Bone, or Individual Number) when the full export/job functions exist.
